pages/sdk/signers/magic.mdx

```diff
@@ -0,0 +1,148 @@
+
+
+import { Tabs, Steps, Callout } from 'nextra/components'
+
+# Magic Signer
+
+In this guide, you will learn how to create a [Magic](https://magic.link/) signer that can be added as a Safe owner and used to initialize any of the kits from the Safe\{Core\} SDK.
+
+Check out the [Safe Signers demo app](https://github.com/5afe/safe-signers) on GitHub to follow along this guide with the completed code.
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- A [Magic account](https://dashboard.magic.link) and an API key.
+
+## Install dependencies
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['npm', 'yarn', 'pnpm']}>
+  <Tabs.Tab>
+    ```bash
+    npm install magic-sdk
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash
+    yarn add magic-sdk
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash
+    pnpm add magic-sdk
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Steps
+
+<Steps>
+
+  ### Imports
+
+  Here are all the necessary imports for this guide.
```

Comment:
We mention that these are all the necessary imports for the guide, but also have more of them [further in the guide](https://github.com/safe-global/safe-docs/pull/562/files#diff-492a82113d7edd1ceff8261eaa67d00c20b154eb7d30645609244bb67b046f09R105). 

We should either change the language of this statement or move all the imports together (both are fine by me).